CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Remove the deprecated API of `0.14.x`. (#3108)
 - [FloatingActionButton] Fix:className now set to root element(#2310)
 - [RaisedButton] Fix:className now set to root element(#3122)
+- [LeftNav] Fix:className now set to root element(#3322)
```

Comment:
@newsiberian Yeah, perfect :+1:. The wording will probably be changed during the writing of the release note. However. that's simpler for us to know ahead of time what the breaking changes are. Thanks!